.github/workflows/pr.yml

```diff
@@ -8,6 +8,10 @@ on:
   push:
     branches:
       - master
+  # automatically uses master https://stackoverflow.com/a/58800550
+  schedule:
+    # daily, 3am UTC
```

Comment:
Worth documenting - what is being gained by running this setup as a cron?

Right now I can't spot the difference between a `master` and `cron` build.

Reply:
Yes, the difference is hidden away in the `cron-matrix` function (jdk+clj matrix build). I'll document that here.